apps/dashboard/src/pages/community/startup-program.tsx

```diff
@@ -190,20 +91,79 @@ const partnersCompanies = [
     maxWidth: "256",
     src: require("../../../public/assets/partners/helika.png"),
   },
-
   {
     maxWidth: "99px",
     src: require("../../../public/assets/partners/play.png"),
   },
+  {
+    maxWidth: "200px",
+    src: require("../../../public/assets/startup-program/ava1anche_logo.png"),
+  },
+  {
+    maxWidth: "150px",
+    src: require("../../../public/assets/startup-program/powered-by-aws-white.png"),
+  },
 ];
 
+const gradientSecond = {
+  src: require("../../../public/assets/startup-program/gradient-2.png"),
+};
+
+const gradientFive = {
+  src: require("../../../public/assets/startup-program/gradient-5.png"),
+};
+
+const cubeTop = {
+  src: require("../../../public/assets/startup-program/cube-top.png"),
+};
+
+const cubeRight = {
+  src: require("../../../public/assets/startup-program/cube-topright.png"),
+};
+
+const cubeBottom = {
+  src: require("../../../public/assets/startup-program/cube-bottom.png"),
+};
+
 const StartupProgram: ThirdwebNextPage = () => {
   return (
```

Comment:
Please add `useForceDarkTheme` hook here to force this page to use the dark mode